typing/typecore.ml

```diff
@@ -1151,6 +1157,8 @@ type pattern_variable =
     pv_id: Ident.t;
     pv_uid: Uid.t;
     pv_mode: Value.l;
+    (* jra: I don't fully understand the difference between mutable_flag and mutability *)
+    pv_mutable: mutable_flag;
```

Comment:
Not quite certain what you're referring to by "mutability" here, but I suspect there isn't a difference - we can discuss.